leptos/src/error_boundary.rs

```diff
@@ -51,6 +51,28 @@ use leptos_reactive::{
 ///       {move || {
 ///   /* etc. */
 /// ```
+///
+/// ## Beginner's Tip: ErrorBoundary Requires Your Error To Implement std::core::Error.
+/// ErrorBoundary requires your `Result<T,E>` to implement `IntoView`.
+/// `Result<T,E>` only implements `IntoView` if E implements `std::core::Error`.
```

Comment:
These types should ideally be links if possible. You can test whether the links work correctly with `cargo doc --no-deps --open` and then check out this page to see if it looks right.